litmus-portal/frontend/src/components/YamlEditor/styles.ts

```diff
@@ -40,87 +58,268 @@ const useStyles = makeStyles((theme: Theme) => ({
 
   editorGrid: {
     overflow: 'auto',
-    maxHeight: '34.5rem',
-    width: '92%',
+    height: '50vh',
+    width: '100%',
+    '&::-webkit-scrollbar': {
+      width: '0.2em',
+    },
+    '&::-webkit-scrollbar-track': {
+      webkitBoxShadow: 'inset 0 0 6px rgba(0,0,0,0.00)',
+    },
+    '&::-webkit-scrollbar-thumb': {
+      backgroundColor: theme.palette.secondary.dark,
+      outline: '1px solid slategrey',
```

Comment:
don't use colors that are not in theme...like slategrey
if this particular color is in design add it to theme properties or use a different shade of grey... mui default theme has a bunch of grey shades